test/org/sagebionetworks/bridge/models/schedules/IntervalActivitySchedulerTest.java

```diff
@@ -70,18 +70,19 @@ public void onceWithoutTimesUsesLocalTime() throws Exception {
         schedule.addActivity(TestConstants.TEST_3_ACTIVITY);
         schedule.setScheduleType(ScheduleType.ONCE);
         schedule.setExpires("P2M");
+        DateTimeZone zone = DateTimeZone.forOffsetHours(-7);
         
         ScheduleContext context= new ScheduleContext.Builder()
                 .withStudyIdentifier(TEST_STUDY)
-                .withInitialTimeZone(DateTimeZone.forOffsetHours(-7))
-                .withEndsOn(ENROLLMENT.plusDays(2)) // in UTC
+                .withRequestTimeZone(zone)
+                .withDaysAhead(2)
```

Comment:
In this test, the initial and the requested timezone were different. Now they are the same. This changes the semantics of the test. Are you sure we're not losing test coverage on this?

Semantically, we should be replacing endsOn.getTimeZone() with requestTimeZone, not initialTimeZone. This seems to be happening a lot in a lot of the other tests as well.